### DISCUSSION TITLE
Statistikdarstellung suboptimal bei zu vielen Betreibern

### DISCUSSION THREAD

**Discussion by FunkoLive** [Traewelling/traewelling#2813] [Ideas / Feature Requests]
Hallo zusammen,

aktuell kann man sich die Statistiken in einem Kreisdiagramm darstellen lassen. Hier gibt es jedoch zwei Probleme:

Erstens gehen bei zu vielen Betreibern gerne die Farben aus.

Zweitens wäre eine Auflistung nach Prozentzahl (also ein Balkendiagramm) schön.

![Screenshot 2024-08-06 at 15-13-16 Statistiken - Träwelling](https://github.com/user-attachments/assets/c9c058f4-3378-4aa4-acc3-9773ff9088c0)

Hier sieht man auf den ersten Blick beispielsweise nicht, ob gerade die S-Bahn München oder DB Regio Mitte den Anteil in rot verursacht.

Ich würde mich freuen, wenn hier mehr Farben und eine Umschaltung auf ein Balkendiagramm eingebaut werden könnten.

Grüße
Felix

**Comment by pascalwittler** [comment]
Mögliche Überschneidung mit #1245 – aber ein Balkendiagramm mit Namen über den Balken wäre echt cool :)
